plugins/pipelineTemplates/create.js

```diff
@@ -0,0 +1,74 @@
+'use strict';
+
+const urlLib = require('url');
+const boom = require('@hapi/boom');
+const schema = require('screwdriver-data-schema');
+const validator = require('screwdriver-template-validator').parsePipelineTemplate;
+const templateSchema = schema.api.templateValidator;
+
+module.exports = () => ({
+    method: 'POST',
+    path: '/pipelineTemplates',
```

Comment:
should we use /pipeline/tempalte instead and move the files under the pipeline folder to maintain the natural structure of the project